.gitignore

```diff
@@ -1,5 +1,18 @@
 *.py[cod]
 
+### Django ###
+*.log
+*.pot
+*.pyc
+local_settings.py
+migrations/
```

Comment:
Nope, see [this]https://docs.djangoproject.com/en/1.11/topics/migrations/.

>The migration files for each app live in a “migrations” directory inside of that app, and are designed to be committed to, and distributed as part of, its codebase. You should be making them once on your development machine and then running the same migrations on your colleagues’ machines, your staging machines, and eventually your production machines.

We decide to push migration files into Systers projects.